### ISSUE TITLE
Stream Bi-Directional, is this possible?

### ISSUE THREAD

**Issue by gmrich** [oxan/esphome-stream-server#51]
Can this stream server work bi-directionally e.g. serial <-> wifi? Its for a modbus like comms system that uses a usb. I would like to expose the usb to wifi.

**Comment by jdwhite** [comment]
Yes, the stream is bi-directional. Can't say if it'll work for your particular situation, but I'm using it to interact with the serial console on a physically remote RPi where connecting cables between the UART and my computer is not practical.
